signUrl/README.md

```diff
@@ -0,0 +1,56 @@
+# Google Cloud Signing URL Sample
+
+This sample shows how to create Cloud CDN signed Url.
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.
+
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.
+
+2.  Follow the instructions here
+    [https://cloud-dot-devsite.googleplex.com/cdn/docs/signed-urls#creatingkeys](https://cloud-dot-devsite.googleplex.com/cdn/docs/signed-urls#creatingkeys)
+    to create the keys that will be used to sign the URLs, upload them and give relevant permissions.
+
+3.  From a Powershell command line, run to see command line arguments.
+    ```
+   PS C:\dev\dotnet-docs-samples\signUrl\SignUrlSample> dotnet run
+   Using launch settings from C:\dev\dotnet-docs-samples\signUrl\SignUrlSample\Properties\launchSettings.json...
+   SignUrlSample 1.0.0
+   Copyright (C) 2018 SignUrlSample
+   ERROR(S):
+   A required value not bound to option name is missing.
+
+  --name          The name of the key to use when encrypting url
+
+  --path          The path to the key to use when encrypting url
+
+  -e              Url expiration in UTC formatted as: YYYY-MM-DDThh:mm:ss
```

Comment:
--name & --path are full but this is short (-e), looks incoherent.

Reply:
Fair point. Fixed so that now we support both short and long forms.